src/domUpdates.js

```diff
@@ -86,40 +85,20 @@ export function displayDailySleepData(sleep) {
 }
 
 export function displayWeeklySleepData(sleep) {
-  const weeklySleep = getWeekly('hoursSlept', sleep, getCurrentDate(sleep));
-  const sleeps = Object.keys(weeklySleep);
-  sleeps.forEach(day => {
-    weeklySleepBox.innerHTML += `<article class="week-day" >
-    <p class="date" >${day.slice(5)}</p>
-    <p class="weekly-ounces">${weeklySleep[day]}h</p>
-    </article>`;
-  });
+  weeklySleepHoursChart(sleep)
 }
 
 export function displayDailySleepQuality(sleep) {
   const dailySleepQuality = getDataByDate(
     'sleepQuality',
-    sleep,
+     sleep,
     getCurrentDate(sleep),
   );
   dailyQualitySleepBox.innerText = `${dailySleepQuality}`;
 }
 
 export function displayWeeklySleepQuality(sleep) {
```

Comment:
The other calls have userWeekly-- passed in can we change 'sleep' to something more descriptive